recipes-app/iot2050-efivarfs-helper/files/iot2050-efivarfs-helper.tmpl

```diff
@@ -144,6 +145,23 @@ def change_efivarfs_immutable_flag(immutable: bool):
             chattr_i(filepath, immutable)
             print("Done.")
 
+def remount_efivarfs(mode):
```

Comment:
I would model the parameter as `readonly=True`, then you don't have to perform any argument checking.

Reply:
But if you foresee any possible other modes, it will be better to have a generic mode.